pkg/registry/certificates/rest/storage_certificates.go

```diff
@@ -60,6 +67,20 @@ func (p RESTStorageProvider) v1Storage(apiResourceConfigSource serverstorage.API
 	return storage, nil
 }
 
+func (p RESTStorageProvider) v1alpha1Storage(apiResourceConfigSource serverstorage.APIResourceConfigSource, restOptionsGetter generic.RESTOptionsGetter) (map[string]rest.Storage, error) {
+	storage := map[string]rest.Storage{}
+
+	if apiResourceConfigSource.ResourceEnabled(certificatesapiv1alpha1.SchemeGroupVersion.WithResource("clustertrustbundles")) {
```

Comment:
Does storage of v1alpha1 types need to be behind a feature gate?  Isn't the type being v1alpha1 (which requires explicit runtime configuration passed to kube-apiserver to serve) enough?

I guess in this case it makes sense (since we don't want to server ClusterTrustBundle objects without the fancy admission checks), but the PR I was following (#109090) doesn't do any additional feature gates.  (It also doesn't guard its changes to node authorizer)

Reply:
Just as we feature gated the alpha `selfsubjectreviews` storage, I would expect us to feature gate this storage as well.

Reply:
Done.